docs/setup/betamocks.md

```diff
@@ -42,6 +60,25 @@ def connection
 end
 ```
 
+However, if you have custom response middleware that changes regularly, it might be worth placing your Betamocks
```

Comment:
Can remove lines 63-81 of this diff

Reply:
Hey @kfrz, is this diff out of date? I don't have those lines anymore in the most recent commit.

Reply:
yep I just pushed the update, sorry for the double-take, thought I could ninja it :)